geth/api/api_test.go

```diff
@@ -0,0 +1,125 @@
+package api
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/status-im/status-go/geth/common"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
Why not `testify/assert`?

Reply:
can you clarify the issue, is it that both assert and require are been used?

Reply:
Oh, sorry, just recognised that you used `require.True()` and similar below. I'm more used to use `assert.True()`. So reacted too fast.